docs/README.rst

```diff
@@ -45,7 +45,33 @@ Please see `How to contribute <https://github.com/saltstack-formulas/.github/blo
 Special notes
 -------------
 
-None
+.. <REMOVEME
+
+Using this template
+^^^^^^^^^^^^^^^^^^^
+
+The easiest way to use this template formula as a base for a new formula is to use GitHub's **Use this template** button to create a new repository. For consistency with the rest of the formula ecosystem, name your formula repository following the pattern ``<formula theme>-formula``, where ``<formula theme>`` consists of lower-case alphabetic characters and numbers.
+
+In the rest of this example we'll use ``example`` as the ``<formula theme>``.
+
+Follow these steps to complete the conversion from ``template-formula`` to ``example-formula``.::
+
+  $ git clone git@github.com:YOUR-USERNAME/example-formula.git
+  $ cd example-formula/
+  $ bin/convert-formula.sh example
+  $ git push --force
+
+Alternatively, it's possible to clone ``template-formula`` into a new repository and perform the conversion there. For example:::
```

Comment:
One too many colons here, as seen on the [rendered page](https://github.com/saltstack-formulas/template-formula/blob/787b3bc9ed55a2f4e9b202695ed5cc9e505e985b/docs/README.rst#using-this-template).

Reply:
Ah yes, I'd misread the reStructuredText docs.